CHANGELOG.md

```diff
@@ -28,6 +28,26 @@ CHANGELOG
 Swift Next
 ----------
 
+* [SE-0299][]:
+
+  It is now possible to use leading-dot syntax in generic contexts to access static members of protocol extensions where `Self` is constrained to a concrete type:
+
+  ```swift
+  public protocol ToggleStyle { ... }
+
+  public struct DefaultToggleStyle: ToggleStyle { ... }
+
+  extension ToggleStyle where Self == DefaultToggleStyle {
+    public static var `default`: Self { .init() }
+  }
+
+  struct Toggle {
+    func applyToggle<T: ToggleStyle(_ style: T) { ... }
```

Comment:
Missing '>'

Reply:
Ugh, thank you! So much for typing everything out...

Reply:
✅